lib/doctest.js

```diff
@@ -322,20 +347,28 @@ function wrap$js(test, sourceType) {
          type === 'ImportDeclaration' ||
          type === 'VariableDeclaration' ?
     test[INPUT].value :
+    // TODO: Why are we enqueing two things if it could just be one structure?
```

Comment:
I believe we are enqueuing one or two things depending on whether the input has associated output.

Reply:
I think it'd be more neat if the input and output would be enqueued as a single structure.

Reply:
Yes, that would be neater. :)

Reply:
I think I'll call it out of scope for this PR. Got to draw the line somewhere.